lib/vmdb/plugins.rb

```diff
@@ -40,6 +40,17 @@ def ansible_content
       end
     end
 
+    def ansible_runner_content
+      @ansible_runner_content ||= begin
+        map do |engine|
+          content_dir = engine.root.join("content", "ansible_tmp")
+          next unless File.exist?(content_dir.join("requirements.yml"))
```

Comment:
My only concern here is that this enforces a requirements.yml.  Is that ok?  If it's only to be used for the purpose of galaxy stuff, then maybe we just need to rename the method, or return both things that have non-galaxy and galaxy content, if that makes sense.

Reply:
requirements.yml came from the galaxy docs https://docs.ansible.com/ansible/latest/reference_appendices/galaxy.html#installing-multiple-roles-from-a-file but you can use that file to seed roles from more than just galaxy.  You can also include other files from within this file.  I don't think you can specify multiple role files, we would have to run ansible-galaxy for each `*.yml` file if we wanted to do that.